paper.md

```diff
@@ -0,0 +1,144 @@
+tools  to facilitate the development of CMake modules are relatively sparse.
+Here we introduce CMinx, a tool for generating API documentation for CMake
+modules. API documentation is arguably one of the most basic elements of a
+software development ecosystem, and it is our hope that CMinx will serve as the
+foundation for a robust CMake development ecosystem.
+
+Anecdotal evidence [@official_solution] indicates that the developers of the
+CMake language (*i.e.*, Kitware) internally write their API documentation
+using reST and Sphinx. Following best practices, this reST documentation
+resides next to the described CMake code. Kitware has also written a
+Sphinx plugin that makes it easy to extract the API documentation as part of a
+normal Sphinx workflow. This Sphinx plugin is distributed with the source code
+for the CMake interpreter and is also available in a GitHub repository mirror
+[@sphinx_plugin]. For completeness, we note that similar Sphinx plugins
+[@official_sphinx_domain; @marco_koch] have been independently developed but
+appear to now be abandoned.
+
+To our knowledge, all of the aforementioned Sphinx plugins simply extract the
+reST API documentation verbatim. Notably, this means the developer is
+responsible
+for manually: adding the function/macro signatures to the documentation, listing
+the function's parameters, and the overall formatting (*e.g.*, titles and
+subtitles, creation of parameter tables, underline/overline consistency). For
+build system developers maintaining one or two CMake modules, these are
+admittedly pretty minor inconveniences; however, for organizations
+maintaining a substantial CMake code base (such as those for exascale programs),
+these "minor inconveniences" can
+impact productivity, particularly when ensuring consistency. CMinx
+differs from previous solutions primarily in three ways. First, CMinx
+understands the grammar of the CMake language, meaning CMinx can automatically
+generate some of the documentation by "reading" the source code. Second, CMinx
+generates static reST files; this decreases the number of stub files developers
+need to maintain and makes it easier for the resulting documentation to be
+used in workflows that do not rely on Sphinx. Finally, CMinx has a CMake API
+to integrate more easily into existing CMake workflows.
+
+Concurrent with the submission of this manuscript, we have also released the
+first production version of CMinx, version 1.0.0. CMinx can be obtained
+from the Python Packaging Index via "`pip install CMinx`". Alternatively, CMinx
+can be used as a CMake module via CMake's "`FetchContent`" command.
+Despite only just releasing 1.0.0, the CMinx GitHub organization
+has already started to see attention and interest from developers not
+affiliated with the authors. So while CMinx might not be considered research
+software, we anticipate CMinx to be a useful productivity tool for the large
+swath of research software that uses CMake as their build system.
```

Comment:
This entire paragraph does not fit into the "Statement of need" and could instead be moved to the end of the "Summary", in my opinion.

Reply:
Done. I moved it to the end of the summary section.